src/features/map/components/Map/Map.jsx

```
@@ -4,16 +4,19 @@ import { DeckGL } from '@deck.gl/react';
 import { GeoJsonLayer, PointCloudLayer, PolygonLayer } from '@deck.gl/layers';
 import { DataFilterExtension } from '@deck.gl/extensions';
 
-import positron from '../../constants/mapStyles/positron.json';
-import no_label from '../../constants/mapStyles/positron_nolabel.json';
+import positron from 'constants/mapStyles/positron.json';
+import no_label from 'constants/mapStyles/positron_nolabel.json';
 
 import * as turf from '@turf/turf';
 import './Map.css';
 
 import { Map } from 'react-map-gl/maplibre';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)